kinds/folder/folder_kind.cue

```diff
@@ -0,0 +1,23 @@
+package kind
+
+name:        "Folder"
+maturity:    "merged"
+description: "A collection of items together"
+
+lineage: seqs: [
+	{
+		schemas: [
+			//0.0
+			{
+				// Unique folder id.
+				uid: string
+
+				// Folder title
+				title: string
+
+				// Description of the folder.
+				description?: string
+			},
```

Comment:
since we're not mapping folders to Namespaces in k8s we'll also need a parent folder ref here to represent nested folders